elements/lisk-p2p/src/p2p.ts

```diff
@@ -126,11 +128,14 @@ const BASE_10_RADIX = 10;
 export const DEFAULT_MAX_OUTBOUND_CONNECTIONS = 20;
 export const DEFAULT_MAX_INBOUND_CONNECTIONS = 100;
 export const DEFAULT_OUTBOUND_SHUFFLE_INTERVAL = 300000;
+export const DEFAULT_PEER_PROTECTION_FOR_NETGROUP = 0.034;
```

Comment:
I added all the code related to secret calculation from @mitsujutsu 's PR #4023  so it will be handled there, I just wanted a secret to initialize the PeerBook